.ci_support/linux_python2.7.yaml

```diff
@@ -1,4 +1,15 @@
+hdf5:
+- 1.10.1
+libnetcdf:
+- '4.5'
+libpng:
+- 1.6.32
+numpy:
+- '1.9'
 pin_run_as_build:
+  numpy:
```

Comment:
sorry, I missed this is my earlier review.  We don't have this in defaults.  It may have been added by CF, or we may have removed it from defaults since we submitted conda-forge-pinning.

It is not good to do numpy here.  Better to do it on the recipe level, not the variant level, as documented at https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-recipe-level

The reason is that when numpy is purely used for its python interface, not its C interface, we don't want to pin it in run depends at all.  If we define the pin_run_as_build value, rather than using pin_compatible, it will always have a run constraint, even when we don't want it (i.e. with numpy's python interface only)

Reply:
It was added by @isuruf in https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/13fba845ba2f39d3b02a8a1678ce24ad3d707a2c

Reply:
PR for removal at https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/44

Reply:
Thanks for finding that out! That is why we need to start using this, so we can finally shake the last few bugs!